docker-compose.yml

```diff
@@ -3,6 +3,7 @@ services:
   oracle:
     build: .
     environment:
+      SLEEP: 3600
```

Comment:
I'd suggest to keep the default value. It's optimal for E2E. Drop the HEAD commit.